src/amo/components/CollectionManager/index.js

```diff
@@ -169,7 +170,7 @@ export class CollectionManagerBase extends React.Component<Props, State> {
     // https://github.com/mozilla/addons-frontend/issues/4635
     // The collectionUpdates state will handle this but it needs
     // to be hooked up the saga.
-    const formIsDisabled = !collection;
+    const formIsDisabled = (!collection || !name.trim());
```

Comment:
I guess the tests pass with this method but I would have thought we needed to test for `.length` here, as `''.trim()` is still truthy.

Reply:
`''.trim()` is falsey, isnt it?

Reply:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/5318732/38173571-db283790-35dd-11e8-9c9e-33f35b61ebb1.png">